.github/workflows/docker.yml

```diff
@@ -1,84 +1,24 @@
 # GitHub actions workflow which builds and publishes the docker images.
 
-name: Build and deploy docker images
+name: Build and deploy Dendrite docker images
 
 on:
-  push:
-    branches: [develop]
```

Comment:
Surely `branches: [dendrite]`?

Reply:
Don't we want to manually build the image? Otherwise we might pull in package updates we don't know, causing unwanted issues.

Reply:
Hmm, because perl doesn't pin dependencies?

Reply:
Both, Perl and distribution packages.